src/WebJobs.Script.WebHost/WebScriptHostManager.cs

```diff
@@ -59,8 +60,8 @@ public class WebScriptHostManager : ScriptHostManager
             IScriptHostFactory scriptHostFactory = null,
             ISecretsRepositoryFactory secretsRepositoryFactory = null,
             HostPerformanceManager hostPerformanceManager = null,
-            int hostTimeoutSeconds = WebScriptHostHandler.HostTimeoutSeconds,
-            int hostPollingIntervalMilliseconds = WebScriptHostHandler.HostPollingIntervalMilliseconds)
+            int hostTimeoutSeconds = RequiresRunningHostAttribute.HostTimeoutSeconds,
```

Comment:
Should this be moved to `Constants` and used by both types?

Reply:
Will do